### ISSUE TITLE
[feature suggestion] List formating

### ISSUE THREAD

**Issue by rene-coty** [vixalien/sticky#2]
It would be great to be able to use lists to format a text, with checkboxes or bullet points...

**Comment by vixalien** [comment]
thank you for the suggestion... I'm currently researching a way to support lists. However, finding a solution that allows checkboxes would be better since it would allow the app to be used as a to-do tracker

**Comment by Nico7as** [comment]
hi! 

Markdown support would allow lots of things, such as lists, checkboxes and such!

That'd be great!

**Comment by TriVoxel** [comment]
Came here to ask for this. Glad to see it is already planned, I cannot wait!

**Comment by vixalien** [comment]
Simple lists have landed as of #107. Checkboxes are probably NOT coming soon...

**Comment by diamondburned** [comment]
Would something like Google Keep's list mode be easier to implement? Under list mode, every line automatically turns into a list, and each line can be moved around by dragging a handle.
This should also be a lot easier to do than trying to fit it into the existing widget, since it should just be a `GtkColumnView`.

**Comment by vixalien** [comment]
> it should just be a `GtkColumnView`.

Yeah, that would be possible, but I'm not trying to implement that just yet. (It would actually be a `GtkListView` or a simple `GtkListBox`)
